src/lib.ts

```diff
@@ -975,3 +1016,10 @@ export function kramcoGuaranteed(): boolean {
       5 + 3 * get("_sausageFights") + Math.pow(Math.max(0, get("_sausageFights") - 5), 3)
   );
 }
+
+export function canWander(): boolean {
```

Comment:
also no idea what this is supposed to be for?

Reply:
this gets used to determine if we are going to fight any wanderers that we might put in guzzlrZones, because that affects our decisions about whether to reject a guzzlr quest. It was suggested to me that we should yoink that out and put it in its own function for legibility, and because it could potentially be useful elsewhere. Just now realizing I should add a little something to make sure you have voters left.